### ISSUE TITLE
Export to Inform 7 not working

### ISSUE THREAD

**Issue by commonterry** [JasonLautzenheiser/trizbort#319]
I am unable to get the "export to Inform 7" function to work in versions 1.5.9.4, .5, and .6. It works fine in .3a, however. It generates a file, but all that's in the file is the name of the map. None of the room code is generated. Thanks!

**Comment by JasonLautzenheiser** [comment]
any chance you can send me your map file (.trizort) so i can test with it?

**Comment by JasonLautzenheiser** [comment]
I've tested this with some maps and the export appears to be working fine in my tests...if you send the particular file you are trying to export, perhaps there is something particular to it that is causing an issue.

Thanks

**Comment by commonterry** [comment]
Sorry - been on the road today and then hanging out with family. Here is
the file - originally created in, I think, 1.2.1.

**Comment by commonterry** [comment]
Also - 1.2.1 can export it as can 1.5.3a, but all versions newer than that
cannot.

Thanks,
Brian

**Comment by JasonLautzenheiser** [comment]
Thanks for the update.   The file doesn't appear to have been attached to your message though.

**Comment by commonterry** [comment]
[map.zip](https://github.com/JasonLautzenheiser/trizbort/files/756457/map.zip)

Sorry I was replying to you from gmail, and I suppose the attachment didn't come through from there. Here it is.

**Comment by JasonLautzenheiser** [comment]
@commonterry Sorry for the delay in looking at this.  I did confirm that this is an issue with older maps.  I'm working on a fix and should have that merged shortly.

**Here is the cause:**  Long ago when I started developing this fork of Trizbort, I added the region feature.  Initially, the default region was blank as it worked well with their being no regions in the older files.  In a later build when I added region support to the Inform 7 export, it did not handle well if all the rooms had no region, which is the case with your map.  When the fix is in place it should handle it appropriately.
